patches/third_party-lit-v3_0-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/lit/v3_0/BUILD.gn b/third_party/lit/v3_0/BUILD.gn
+index 850660bc45180953e32b0b1d38ace3c5263d56dc..e91080bbf82d547ebb27b06dd62824e0895a37e2 100644
+--- a/third_party/lit/v3_0/BUILD.gn
++++ b/third_party/lit/v3_0/BUILD.gn
+@@ -37,6 +37,7 @@ ts_library("build_ts") {
+     "//ui/webui/resources/cr_components/most_visited:build_ts",
+     "//ui/webui/resources/cr_components/theme_color_picker:build_ts",
+     "//ui/webui/resources/cr_elements:build_ts",
++    "//chrome/browser/resources/settings:build_ts", "//brave/ui/webui/resources:build_ts"
```

Comment:
this does not follow patching standards, you should have `visibility += brave_lit_build_ts_visibility`

Reply:
cc: @fallaciousreasoning

Reply:
Fixed!

Reply:
@fallaciousreasoning I'm not sure what's going on, but it seems that `npm run sync` always reapplies this particular patch.

Reply:
@cdesouza-chromium sorted this out.